src/components/DashStat.container.js

```diff
@@ -12,17 +12,20 @@ import PeopleOutlineOutlinedIcon from '@material-ui/icons/PeopleOutlineOutlined'
 import apiPlanters from '../api/growers';
 import api from '../api/treeTrackerApi';
 import FilterModel from '../models/Filter';
+import { useKeycloak } from '@react-keycloak/web';
 
 function DashStatTotalCaptures(props) {
   const activeFilter = new FilterModel({
     active: true,
   });
 
   const [total, setTotal] = useState(null);
-
+  const {
+    keycloak: { token },
+  } = useKeycloak();
   const getTotal = async () => {
     console.log('-- dash getTotal');
-    const { count } = await api.getCaptureCount(activeFilter);
+    const { count } = await api.getCaptureCount(activeFilter, token);
```

Comment:
I don't think the component should need to know about `token` and be responsible for retrieving it.
This may be better placed in the API function.

Reply:
@TANguyen1893 is using a hook for this, so it's easy to get from the component side.

Reply:
To make this structure better, maybe we can use custom hook to initial API object